spec-lang/dune-project

```diff
@@ -0,0 +1 @@
+(lang dune 1.6)
```

Comment:
Don't include this file -- it's a build artifact also. Maybe add it to the gitignore?

Reply:
okay removed it